session.go

```diff
@@ -418,11 +419,23 @@ func (s *session) WriteBytes(pkg []byte) (int, error) {
 		return 0, ErrSessionClosed
 	}
 
-	lg, err := s.Connection.send(pkg)
+	totalSize := len(pkg)
```

Comment:
totalSize := len(pkg)
        pkgArray := make([][]byte, 0, totalSize / maxPacketLen + 1)
	writeSize := 0
	for totalSize >= maxPacketLen {
                 pkgArray = append(pkgArray, pkg[writeSize:(writeSize + maxPacketLen)])
		totalSize -= maxPacketLen
		writeSize += maxPacketLen
	}
        pkgArray = append(pkgArray, pkg[writeSize:])
        lg, err := s.Connection.send(pkgArray)
        if err != nil {
	    return writeSize, perrors.Wrapf(err, "s.Connection.Write(pkg len:%d)", len(pkg))
        }

Reply:
address